### PR TITLE
Detection mismatches

### PR DESCRIPTION
**Pull request by mimmi20**
With this PR fixes most of detection mismatches when using the native get_browser function.

This PR requires PR #1746 to be merged before.
This PR also requires PR #1765 to be merged before because there are similar changes.

### CODE REVIEW

**Comment by jaydiablo** [comment]
@mimmi20 Please rebase and resolve conflicts when you have a chance :)

File: tests/issues/issue-000-invalids.php [hunk 1]
```diff
@@ -1,60 +1,395 @@
 <?php
```
Comment (jaydiablo) [inline tests/issues/issue-000-invalids.php]: Please revert this file again :)

File: tests/issues/issue-000-invalid-versions.php [hunk 2]
```diff
@@ -1,593 +1,1558 @@
 <?php
```
Comment (jaydiablo) [inline tests/issues/issue-000-invalid-versions.php]: Please revert this file again. :)

File: tests/issues/issue-900.php [hunk 3]
```diff
@@ -38638,10 +38638,139 @@
         'standard' => true,
         'full' => true,
     ],
+    'issue-900-ARU' => [
```
[the remark this answers is not in the document]
Comment (mimmi20) [reply tests/issues/issue-900.php]: These teste were created with #900, but for a non-mobile. You moved these tests to the `invalid versions`. As get_browser detectes these as mobiles in desktop mode I restored the old tests.

File: tests/issues/issue-961.php [hunk 4]
```diff
@@ -867,21 +867,6 @@
         'standard' => false,
         'full' => true,
     ],
-    'issue-961-Q (standard)' => [
```
[the remark this answers is not in the document]
Comment (mimmi20) [reply tests/issues/issue-961.php]: the test is restored.

**Pull request merged**